server.js

```diff
@@ -0,0 +1,73 @@
+var express = require('express');
+
+import { request, abort, } from 'request-promise';
+
+const port = process.env.PORT || 1337;
+
+const app = express();
+
+async function incoming2outgoingPayload(payload) {
+  import jsontoxml from 'jsontoxml';
+  const itemsToXML = {}
+  for(let i = 0; i <= payload.items.length; i++) {
+    const startDate = new Date(payload.items[i].start);
+    const endDate = new Date(payload.items[i].end);
+    itemsToXML[`vacation-${startDate.toString()}-${endDate.toString()}`] = payload.items[i].approved == true ? true : false;
```

Comment:
nice work with the template strings here, I like using them too! makes this easier to read.

quick suggestion though, looks like you're expecting `approved` to be a boolean already, so probably no need for the ternary operator here.

```suggestion
    itemsToXML[`vacation-${startDate.toString()}-${endDate.toString()}`] = payload.items[i].approved;
```